packages/indicators/src/__tests__/__integration/integration.test.ts

```diff
@@ -38,13 +40,31 @@ describe('Integration tests', () => {
     await dataBroker.close();
   });
 
-  runIntegrationTests({
-    config: indicatorApiFixtures,
-    it: runTestCase,
+  describe('Indicator API features', () => {
+    const { setup, testCases } = indicatorApiFixtures;
```

Comment:
@edmofro I was thinking that the runIntegrationTest functionality was too abstracted with unclear benefits. I feel it's very beneficial for a developer (including myself) to see the familiar nested describe/it structure in a test, be it of unit or integration flavour.

Reply:
I totally agree! Thanks for again being willing to throw away your own code 🙏